### PR TITLE
Big Pharma

### PR DESCRIPTION
**Pull request by W-Floyd**
| Original icon | Circle icon | Square icon |
| --- | :-- | :-- |
|  original |  circle | square |

### CODE REVIEW

**Comment by palob** [comment]
Thanks for your contribution.
The symbol design is nice yet too big. Can you shrink it down to a height of 24px, 26px (or 28px)?

**Comment by W-Floyd** [comment]
Yup, had thought that'd need to happen. I'll get onto it later today.

**Comment by W-Floyd** [comment]
Before and after:
image

**Comment by palob** [comment]
What's the new size?

**Comment by W-Floyd** [comment]
26 ish pixels high (because the outline of the beaker is a stroke at present, the corner bumps it out a bit)

**Comment by palob** [comment]
Alright, if all vertical lines are aligned to the grid and the strokes are converted to paths it's good to go.

You need to reopen this PR without any merged commits before we can merge it though.
(It's not enough to revert the commits and reapply them, there must not be any in the git history. Otherwise buggy Launchpad will break the PPA.)

**Comment by W-Floyd** [comment]
Okay, I'll convert the strokes then.
I won't pretend to be that good in git, but would re-writing history work? I'm happy to re-open, just curious.

**Comment by palob** [comment]
Re-writing is OK - in case no trace is left. We had to do this in the past.

**Comment by W-Floyd** [comment]
Okay, I'll give that a go.

**Comment by W-Floyd** [comment]
Here's the updated version, I believe the commit history is okay now.
image 
image

**Comment by W-Floyd** [comment]
iconified_steam_icon_344850

**Review by palob — APPROVED** [review]

File: icons/circle/48/steam_icon_344850.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ <g transform="translate(-2,-54)">
+  <g transform="translate(1,1)"></g>
+ </g>
+ <path d="m15.75 30" fill="none"/>
+ <path d="m20.25 30" fill="none"/>
+ <g fill-rule="evenodd">
+  <path d="m20 10.586l-1.4141 1.4141 0.70703 0.70703 1.707 1.707v7.4121l-5.4199 15.174h18.84l-5.4199-15.174v-7.4121l1.707-1.707 0.70703-0.70703-1.4141-1.4141-0.70703 0.70703-2.293 2.293v8.5879l4.582 12.826h-13.164l4.582-12.826v-8.5879l-2.293-2.293-0.70703-0.70703zm3.8574 14.414l-2.8574 8h8l-2.8574-8h-2.2852z" color="#000000" color-rendering="auto" dominant-baseline="auto" image-rendering="auto" opacity=".1" shape-rendering="auto" solid-color="#000000" style="font-feature-settings:normal;font-variant-alternates:normal;font-variant-caps:normal;font-variant-ligatures:normal;font-variant-numeric:normal;font-variant-position:normal;isolation:auto;mix-blend-mode:normal;shape-padding:0;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-orientation:mixed;text-transform:none;white-space:normal"/>
```
Comment (palob) [inline icons/circle/48/steam_icon_344850.svg]: There's some strange stuff here. Can you please save the files again?

Comment (W-Floyd) [reply icons/circle/48/steam_icon_344850.svg]: Tried just now, didn't change that section... I'll see if there's anything weird going on that I can find.

**Comment by W-Floyd** [comment]
Still looks the same, but I think that takes care of the weirdness.

**Review by palob — CHANGES_REQUESTED** [review]
Theres still strange paths, e.g. 
```xml
<path d="m20 10.586-1.4141 1.4141 2.414 2.414v7.4121l-5.4199 15.174h18.84l-5.4199-15.174v-7.4121l2.414-2.414-1.4141-1.4141-3 3v8.5879l4.582 12.826h-13.164l4.582-12.826v-8.5879zm3.8574 14.414-2.8574 8h8l-2.8574-8z" dominant-baseline="auto" style="color-rendering:auto;color:#000000;dominant-baseline:auto;fill-rule:evenodd;font-feature-settings:normal;font-variant-alternates:normal;font-variant-caps:normal;font-variant-ligatures:normal;font-variant-numeric:normal;font-variant-position:normal;image-rendering:auto;isolation:auto;mix-blend-mode:normal;opacity:.1;shape-padding:0;shape-rendering:auto;solid-color:#000000;text-decoration-color:#000000;text-decoration-line:none;text-decoration-style:solid;text-indent:0;text-orientation:mixed;text-transform:none;white-space:normal"/>
```

There are several irrelevant variables at the end.

**Comment by W-Floyd** [comment]
Huh... Maybe I should just redraw it?

**Comment by palob** [comment]
Yeah, maybe start with a fresh baseplate and paste symbol+shadow in.

**Comment by W-Floyd** [comment]
I tried that, no luck. I tried re-drawing from scratch, using the template. Still no luck! I am stumped :P

**Comment by W-Floyd** [comment]
iconified_steam_icon_344850

**Comment by W-Floyd** [comment]
I have tentatively removed the extra stuff in the paths. Reoptimizing doesn't change it back, so I'm happy with it.

**Review by palob — APPROVED** [review]

**Review by palob — APPROVED** [review]

**Comment by palob** [comment]
Fixed #3040

**Pull request merged**